tests/test_result_to_dict.py

```diff
@@ -200,6 +202,21 @@ def test_callback(self):
         data = result_to_dict(json_value_without_str_obj, callback=convert_return_none)
         self.assertDictEqual(data, json_value_without_str_obj)
 
+    def test_pymongo_to_dict(self):
+        with connect_to_mongo().get() as client:
+            collection = client.testing_db.example
+            data_not_converted = collection.find()

```

Comment:
again. this is testing nothing if the cursor is empty. 
please move this to the end of the test function

then add 3 elements 
then assert they are there

Reply:
Hi, there are three assert statements written for this. We are checking that if the data is converted to a list it should be an instance of a list, if the data is being converted  to a list it should not be an instance of a cursor object and if the data is being converted to a list it should not be equal to the data_not _converted which is a cursor object the collection returns by default. No, matter if the cursor is empty or not these testcases will work because we are not checking CRUD here. we are checking if the data is being converted to list or not. Thank you